### PR TITLE
Add Governed Repo Templates

### PR DESCRIPTION
**Pull request by sheylatrudo**
**Reason for Change**:

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] relevant PR labels added

**Notes**:

### CODE REVIEW

**Comment by jpayne3506** [comment]
We may want to test that this works correctly by merging this to a test branch and then creating a PR against it.

**Comment by sheylatrudo** [comment]
> We may want to test that this works correctly by merging this to a test branch and then creating a PR against it.

This is just a draft PR right now. Created this to test on some of the builds.

File: .pipelines/pipeline.yaml [hunk 1]
```diff
@@ -1,23 +1,60 @@
-pr:
```
Comment (jpayne3506) [inline .pipelines/pipeline.yaml:1]: Why remove the pr trigger and leave the merge queue trigger? I think it should be the other way around.

File: .devops/OneBranch.PullRequest.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+trigger:
```
Comment (jpayne3506) [inline .devops/OneBranch.PullRequest.yaml:1]: So, this yaml should run our full E2E whenever we add it to the merge queue?

**Pull request closed without merging**